src/tests/components/searchForm.test.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import SearchForm from "../../components/SearchForm";
+import setSearchText from "../../components/App";
+import handleCitySearch from "../../components/App";
```

Comment:
These two lines should be removed. It looks like youre trying to add in functions from another component, but this isnt possible as these functions are not marked with the `export` keyword. Besides, for tests, you do not need to do this as you want to recreate all things passed into the test component using a set of valid test props:
```
const validProps = {
  searchText: "",
  setSearchText: jest.fn,
  onSubmit: jest.fn,
};
```

You can then pass these into the test components with something like `{validProps.searchText}`